modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Settings.tsx

```diff
@@ -0,0 +1,197 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import {Option, Picker, Text} from '@clayui/core';
+import DropDown from '@clayui/drop-down';
+import {ClayToggle} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import {navigate} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import {IFDSViewSectionProps} from '../FDSView';
+
+const NOT_CONFIGURED_VIEW_MODE = {
+	id: 0,
+	label: Liferay.Language.get('configure-new-layout'),
```

Comment:
I know what figma says, however, for consistency sake, I'd use the same term "visualization mode" in the language key and its value, kind of `configure-new-visualization-mode`.

Reply:
I´ve received confirmation from Uge that we can use "Go to Visualization Modes" instead.

Reply:
Thanks Dani